target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -146,6 +156,21 @@ define Device/netgear_wax620
 endef
 TARGET_DEVICES += netgear_wax620
 
+define Device/netgear_wax630
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Netgear
+	DEVICE_MODEL := WAX630
+	DEVICE_DTS_CONFIG := config@hk01
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	SOC := ipq8074
+	IMAGES := ui-factory.tar factory.ubi sysupgrade.bin
```

Comment:
Is the `factory.ubi` ever used?

Reply:
Yes.
`# ubiformat /dev/mtd18 -y -f /tmp/openwrt-qualcommax-ipq807x-netgear_wax630-squashfs-factory.ubi`